internal/state/game.go

```diff
@@ -0,0 +1,10 @@
+package state
+
+type Game struct {
+   players  []Player
+   enemies  []Enemy
+   goals    []Goal
```

Comment:
Goals? not Crystals? Am I mistaken that the objective is to get all crystals?

Reply:
yeah, i suppose I was just trying to make the name generic.  like, when we skin it, enemies might become "ghosts", in the same way that goals are crystals